lib/Sema/TypeChecker.h

```diff
@@ -549,7 +549,7 @@ class TypeChecker final : public LazyResolver {
   std::vector<AbstractFunctionDecl *> definedFunctions;
 
   /// Declarations that need their conformances checked.
-  llvm::SmallVector<Decl *, 8> ConformanceContexts;
```

Comment:
This particular change probably doesn't do much, because there's only really one TypeChecker instance that lives forever.